cordapp/src/test/resources/net/corda/plugins/CorDappWithInfoAll.gradle

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id 'java'
+    id 'net.corda.plugins.cordapp'
+}
+
+apply from: 'repositories.gradle'
+
+jar {
+    baseName 'test-cordapp'
+}
+
+cordapp {
+    targetPlatformVersion target_version_arg.toInteger()
+    minimumPlatformVersion min_platform_version_arg.toInteger()
+    info {
```

Comment:
Surely specifying `info` alongside `contract` or `workflow` should not be allowed. Also if you specify the top level `targetPlatformVersion` or `minimumPlatformVersion` with `info`. You have to have extra checks to make sure the values are consistent.

Reply:
Fixed.
New top-level, `contract` and `workflow` attributes will always take precedence over older (deprecated `info` object items)